### PR TITLE
Create 1.0.2-msbuild-preview image

### PR DESCRIPTION
**Pull request by natemcmaster**
Creates a `aspnetcore-build:1.0.2-msbuild-preview` image.

Identical to the `aspnetcore-build:1.0.1-msbuild` image but with dependency lifted to `microsoft/dotnet:1.0.2-sdk-msbuild`.

cc @SteveLasker @MichaelSimons @glennc

### CODE REVIEW

File: 1.0.2/build-msbuild/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# TODO pull from microsoft/dotnet when these become available
+FROM microsoft/dotnet-nightly:1.0.2-sdk-msbuild
```
Comment (MichaelSimons) [inline 1.0.2/build-msbuild/Dockerfile]: I don't think we should be producing a 1.0.2-sdk-msbuild image.  This workaround should be contained as much as possible IMO.  I would go so far as ask if you can just retag 1.0.1 as 1.0.2 and not call it out in the readme?

Comment (natemcmaster) [inline 1.0.2/build-msbuild/Dockerfile]: So, there won't be a `microsoft/dotnet:1.0.2-sdk-msbuild`? It shows up in the nightly repo. 

> not call it out in the readme

:+1: yeah, we'll try to fix VS templates instead.

Comment (MichaelSimons) [inline 1.0.2/build-msbuild/Dockerfile]: That was a mis-tagged image and deleting tags is currently broken in DH.  You should be referring to the readme for the officially supported images.

Comment (natemcmaster) [inline 1.0.2/build-msbuild/Dockerfile]: Okay. Cool. Then yes, these changes are not necessary.

**Comment by natemcmaster** [comment]
Closing. We'll make aspnetcore-build:1.0.2-msbuild-preview an alias for aspnetcore:1.0.1-msbuild.

**Pull request closed without merging**